terrarium/terrarium.py

```diff
@@ -558,12 +563,17 @@ def after_install(options, base):
         from pip.baseparser import create_main_parser
         main_parser = create_main_parser()
         c = InstallCommand(main_parser)
-    reqs = shlex.split(' '.join(REQUIREMENTS), comments=True)
-    options, args = c.parser.parse_args(reqs)
+
+    fd, file_path = tempfile.mkstemp()
+    with os.fdopen(fd, 'w') as f:
+        f.write('\n'.join(REQUIREMENTS))
+    options, args = c.parser.parse_args(['-r', file_path])
```

Comment:
So, to make sure I'm understanding this correctly:

Instead of parsing the requirements file ourselves and passing each one as an argument to the pip command, we're now using `-r` to use pip's own parsing functionality, which transparently handles comments. Saavy?